terraform/modules/gke/gke.tf

```diff
@@ -14,38 +14,42 @@ variable "machine_type" {
   type = string
 }
 
-data "google_compute_zones" "available" {}
-
 resource "google_container_cluster" "cluster" {
-  name = "${var.resource_prefix}-cluster"
+  provider = google-beta
```

Comment:
I think you should leave a comment explaining what requires `google-beta` in this block (so that, as it gets moved into GA in the plugin we can switch back to non-beta).

Reply:
I moved this line under the `networking_mode = "VPC_NATIVE"` line so it's more obvious why we opt into that provider.